src/scene/forward-renderer.js

```diff
@@ -3106,18 +3089,16 @@ class ForwardRenderer {
                 this._sortTime += now() - sortTime;
                  // #endif
 
-                objects = layer.instances;
-                visible = transparent ? objects.visibleTransparent[cameraPass] : objects.visibleOpaque[cameraPass];
+                let objects = layer.instances;
+                let visible = transparent ? objects.visibleTransparent[cameraPass] : objects.visibleOpaque[cameraPass];
 
                 // Set the not very clever global variable which is only useful when there's just one camera
                 this.scene._activeCamera = camera.camera;
 
                 // Set camera shader constants, viewport, scissor, render target
                 this.setCamera(camera.camera, renderAction.renderTarget);
 
-                // #ifdef PROFILER
-                draws = this._forwardDrawCalls;
-                // #endif
+                const draws = this._forwardDrawCalls;
```

Comment:
What's the thinking behind accumulating draw calls in all builds instead of just PROFILER build?

Reply:
they already get accumulated in all builds inside the actual drawing code .. so this just adds them on top level, which runs only few times a frame .. so low cost benefit for the cost we already pay for. Users were asking for some stats to be available in all builds.

Reply:
OK, sounds good.